source/Calamari.Tests/Fixtures/Integration/Packages/DockerImagePackageDownloaderFixture.cs

```diff
@@ -68,8 +68,8 @@ public void PackageWithoutCredentials_Loads()
         [RequiresDockerInstalled]
         public void DockerHubWithCredentials_Loads()
         {
-            const string privateImage = "octopusdeploy/octo-prerelease";
-            var version =  new SemanticVersion("7.3.7-alpine");
```

Comment:
This should have already been changed in main, unsure why it's showing in this diff